velox/exec/ExchangeClient.cpp

```diff
@@ -140,6 +140,9 @@ void ExchangeClient::request(const RequestSpec& requestSpec) {
           RequestSpec requestSpec;
           {
             std::lock_guard<std::mutex> l(queue_->mutex());
+            if (closed_.load()) {
```

Comment:
Does this need to be checked while holding a lock?

Reply:
It looks like this is the actual change, and its purpose is: if the `ExchangeClient` has been closed, it will no longer continue to send new requests? right?

we can just use `if(closed_)` since it is under the protection of `queue_->mutex()`.